kie-maven-plugin/src/it/kie-maven-plugin-test-kjar-7-exec-model/src/test/java-filtered/org/kie/maven/plugin/ittests/KJarWithDMNIntegrationTestIT.java

```diff
@@ -52,6 +52,10 @@
     public void testComplexDMNModel() throws Exception {
         final URL targetLocation = KJarWithDMNIntegrationTestIT.class.getProtectionDomain().getCodeSource().getLocation();
         final KieSession kieSession = ITTestsUtils.getKieSession(targetLocation, GAV_ARTIFACT_ID, GAV_VERSION, KBASE_NAME);
+
+        // verify DMN file is present
+        assertThat(getClass().getClassLoader().getResourceAsStream("META-INF/kie/dmn")).isNotNull();
```

Comment:
What about use [DMNRuleClassFile.RULE_CLASS_FILE_NAME](https://github.com/kiegroup/drools/blob/master/kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/core/compiler/execmodelbased/DMNRuleClassFile.java#L15) instead of copy the string?